util.c

```diff
@@ -5737,6 +5737,52 @@ S_xs_version_bootcheck(pTHX_ SSize_t items, SSize_t ax, const char *xs_p,
     }
 }
 
+/*
+=for apidoc api_version_assert
+
+Used by the PERL_API_VERSION_CHECK macro to compare the perl the
+object was built with and the perl that C<libperl> was built with.
+
+This can be used to ensure that these match and produces a more
+diagnosable than random crashes and mis-behaviour.
```

Comment:
There seems to be a word or two missing after `diagnosable`.  `diagnosable behavior`, perhaps?